pyccel/ast/core.py

```diff
@@ -4327,6 +4395,62 @@ def execute(self):
 
 # TODO: Should Declare have an optional init value for each var?
 
+class FuncAddressDeclare(Basic):
```

Comment:
Missing class docstring

Reply:
@yguclu any idea why these are not raising codacy errors?

Reply:
@EmilyBourne Not really